### ISSUE TITLE
Evaluate Hydraulic options for tractor design

### ISSUE THREAD

**Issue by goscommons** [goscommons/Tractor#5]
[See the lifetrac hydraulics design by Open Source Ecology](http://opensourceecology.org/wiki/File:Ostrac_hydraulics.jpg)
[See also the hydraulics intent of OSE](http://opensourceecology.org/wiki/LifeTrac_hydraulics_primer)
![image](https://user-images.githubusercontent.com/34537534/39603103-83823154-4f27-11e8-8b48-6ac0c88fc2ff.png)

**Comment by polemidis** [comment]
He have to check the hydraulics more closely because from my experience with Marcin I no longer trust him in hydraulics at all :)

**Comment by vinaybhajantri** [comment]
Allis Chalmers model has no hydraulics, the main role of hydraulics in the example above was to actuate the implements and I guess steering, I understand this is important since it is a 4WD. 

But we can keep steering and actuation mechanical, right? Is there anything I'm missing about hydraulics?

**Comment by jurra** [comment]
What is the pros and cons of having hydraulics steering? Many projects I have seen adopt hydraulics in their tractors. I guess it makes everything smoother? The cultitrack uses hydraylics, Oggun as well, Tilmor tractor also has hydraulics. I am not an expert on this topic so I am only able to post questions:

**Which reference models should we start with old Allis Chalmers G models, or modernized versions like the ones put in the references?**

**Isn't it hydraulics used in tractors regardless of the models? I have seen also Open Source Ecology using those, in a different way though?**

**Do we need hydraulics for implement attachment? I think yes.** Then why not to consider hydraulics also for steering, could be with a cylinder, not necessarily with the model above.

**Comment by polemidis** [comment]
I think that before going to the hydraulics discussion you need to answer what exactly do you want to machine to do?? This will direct all the questions above.
Do you want to have Front End Loader (FEL) (mostly either bucket, grapple)? How about rear attachments on the 3point hitch, then Power Take Off(PTO)??
The Allis Chalmer  has only middle attachments, which nowdays are almost obsolete, other than a mower.

**Comment by vinaybhajantri** [comment]
Okay, I assume **power steering** is a hard requirements here. We have two options- hydraulics and electric. 

### Hydraulic
Pros: 
1. Cheaper
2. More robust, and easy to repair

Cons: 
1. High parasitic power consumption; meaning it takes power from engine 
2. Complex from component stand point
3. Hydraulic fluid needs replacement

### Electric
Pros: 
1. Simpler component construction and lighter
2. Engine power is not compromised
3. If steering fails, there is a backup mechanical steering always

Cons: 
1. Expensive
2. Diagnosing for repair needs expertise

**Comment by vinaybhajantri** [comment]
I agree with @polemidis . Taking a decision on this will also help a lot in defining other parts of the vehicle especially the drivetrain unit. But I lack knowledge in this domain, so I'm going to read about it a little. 

Do you have any idea about what small-scale farmers usually want/need?

**Comment by polemidis** [comment]
Any farm needs somehing to move heavy stuff around. (soil, compost, fertilizers, posts etc) so here it is the FEL. Since we have the grapple already, thats another vote for the FEL
Also 99% of the implements need PTO at the back. Tillers, post hole digger, cultivator, disks, box blade, bed shaper, transplanter, plastic roller, are just a few.

**Comment by jurra** [comment]
## Can we find a tractor with expired patents that has a front loader? 
I have been looking for it couple of days ago in google patents. Is not enough to find the pantents, we need to make sure that the model was actually successful commercially and used in farms.

It makes a lot of sense, all the reflections. I would add a reflection in terms of strategy and community traction, as well as investment traction. The good strategy of many of open source projects, is to clone something that works, and then improve. The prusa model, as other machines, came from hacking expiring patents.

At this point I would consider this first v1 of an Allis Chalmers, as a project to learn, build capacity, community and reaching a deliverable product fast. By having a modern clone of existing Allis Chalmers that are being commercialized, and don't hold patents, we have a product, that others can hack, and improve. I think this criterion is also important. Where we are, what we can do at the moment, vs what can we do later, with more people, expertise and capital. 

This is my personal answer to @polemidis question. What is the machine for? Great question.

Is not just about the tractor. Is also about community building, attracting investors, and parties interested in collaborating around an open source tractor design, that can evolve in new releases. 

So as a general remark, I suggest to pick something that works for someone and has no patent related issues, to start leveraging our project. Start with something that works.

**Comment by jurra** [comment]
Jannes and I were talking about hydraulics, gearbox, etc. I really have no clear idea of this, but found this reference which is interesting: [Mechanical, electrical or hydraulic transmission](http://www.csbe-scgab.ca/docs/journal/2/2_1_24_ocr.pdf). This issue is still open, is good to have a clear stand on the topic to close it.
